pages/_document.tsx

```diff
@@ -57,6 +57,7 @@ class MyDocument extends Document {
       <Html lang='en'>
         <Head>
           <link href='https://fonts.googleapis.com/css2?family=Rubik:wght@400&display=swap' rel='stylesheet' />
+          <link href='https://fonts.googleapis.com/css2?family=Teko&display=swap&text=0123456789' rel='stylesheet' />
```

Comment:
Would prefer if we can figure out how to conditionally include this as not everyone will be using this font

Reply:
Also, just to confirm, what is license for this font?

Reply:
SIL OFL: https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL

Reply:
Couldn't find a way to conditionally add this link so for now maybe it's fine to leave it like this. Don't think this will impact performance that much